### ISSUE TITLE
SP Cientista de Dados na ASSESSO INFORMÁTICA

### ISSUE THREAD

**Issue by Amanda-Costa-Psi-HR** [datascience-br/vagas#104]
## Descrição da vaga
A ASSESSO está em busca de Cientista de Dados para realizar pesquisas, encontrar insights e propor soluções de data science/ machine learning para grandes clientes. Também participa de discussões de desenvolvimento de PoCs (Provas de Conceito) e	Interação com o time de desenvolvedores para a integração de modelos de Machine Learning com o produto (DataCare).

## Local
Híbrido: Alphaville/ SP.

## Diferenciais
- Gympass
- Parceria com workplace de seguros
- Descanso remunerado anual.

#### Obrigatórios:
- Experiência em ferramentas de Machine Learning 
- Experiência na manipulação e extração informações de repositórios de dados estruturados e não-estruturados
- Experiência em análise exploratória (capacidade de analisar um dataset e apresentar hipóteses)
- Experiência na utilização de bibliotecas de Machine Learning em Python
- Experiência em técnicas de modelagem preditiva e descritiva
- Conhecimentos de estatística
- Conhecimento em bancos de dados diversos: Bancos de dados SQL e NoSQL
- Experiência no uso de bibliotecas Open source para análise, exploração e uso de modelos preditivos
- Experiência no uso de ferramentas em plataformas clouds (Google, Azure e AWS)

#### Desejáveis:
- Experiência com ferramentas de Gradient Boosting. (XGBoost, LGBM, etc)
- Experiência com TensorFlow
- Experiência com NLP

## Contratação
- PJ a combinar

## Nossa empresa
A Assesso é líder no segmento de Qualidade e Governança de Dados, possui certificação do MIT e uma ferramenta própria chamada Datacare, que faz a gestão de dados dos nossos clientes. A nossa missão é ajudar o cliente a obter as informações assertivas e exatas, incluindo o enriquecimento com dados, estudos de probabilidades e tendências que eles possam necessitar. Para mais detalhes, acesse: assesso.com.br

## Como se candidatar
Por favor envie um email para rh@assesso.com.br com seu CV anexado - enviar no assunto: Vaga Cientista de Dados

Labels:
PJ
Sênior
